ios/Podfile

```diff
@@ -1,50 +1,22 @@
-# require_relative '../node_modules/react-native/scripts/react_native_pods'
-# require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
-
-# platform :ios, '11.0'
-
-# target 'rnnsimple' do
-#   config = use_native_modules!
-
-#   use_react_native!(:path => config["reactNativePath"])
-
-#   target 'rnnsimpleTests' do
-#     inherit! :complete
-#     # Pods for testing
-#   end
-
-#   # Enables Flipper.
-#   #
-#   # Note that if you have use_frameworks! enabled, Flipper will not work and
-#   # you should disable these next few lines.
-#   # use_flipper!
-#   # post_install do |installer|
-#   #   flipper_post_install(installer)
-#   # end
-# end
-
-# target 'rnnsimple-tvOS' do
-#   # Pods for rnnsimple-tvOS
-
-#   target 'rnnsimple-tvOSTests' do
-#     inherit! :search_paths
-#     # Pods for testing
-#   end
-# end
-
-
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 platform :ios, '11.0'
+install! 'cocoapods', :deterministic_uuids => false
 
 target 'rnnsimple' do
   config = use_native_modules!
 
+  # Flags change depending on the env values.
+  flags = get_default_flags()
+
   use_react_native!(
     :path => config[:reactNativePath],
     # to enable hermes on iOS, change `false` to `true` and then install pods
```

Comment:
Worth changing the comment here to this:
# By default, Hermes is disabled on Old Architecture, and enabled on New Architecture.
# You can enabled/disable it manually by replacing flags[:hermes_enabled] with true or false.